mock2/private/style/scss/article/article.scss

```diff
@@ -0,0 +1,73 @@
+.article {
+  max-width: 420px;
+  margin: 0 10px;
+  color: $mainTextColor;
+  font-size: 14px;
+  font-family: "Playfair Display", serif;
+  font-weight: 400;
+
+  &_introduction {
+    max-width: 100%;
+    margin: 0;
+  }
+
+  &__image {
+    height: auto;
+    width: 100%;
+    margin-bottom: 32px;
+  }
+
+  &__image_introduction {
+    max-height: 509px;
+    margin-bottom: 99px;
+  }
+
+  &__label {
+    display: block;
+    margin-bottom: 28px;
+    color: $labelTextColor;
```

Comment:
I'd suggest you to name color variables like: `color-primary` or `color-secondary`